cmake/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ string(TOUPPER "${CMAKE_BUILD_TYPE}" BTYPE)
 # this is fast, so check for it all the time
 message(STATUS "Running check for auto-generated files from make-based build system")
 file(GLOB SRC_AUTOGEN_FILES ${LAMMPS_SOURCE_DIR}/style_*.h)
+file(GLOB SRC_AUTOGEN_FILES ${LAMMPS_SOURCE_DIR}/packages_*.h)
```

Comment:
Isn't that overwriting the content of `SRC_AUTOGEN_FILES` from the line above?

Reply:
yes it does. shouldn't be a big issue, but needs to be fixed. will sort it out ASAP.

Reply:
done